crates/bevy_asset/src/processor/mod.rs

```diff
@@ -1,3 +1,25 @@
+//! Asset processing in Bevy is a framework by which artist-authored assets can be automatically prepared for use in game.
+//!
+//! You can think of the asset processing system as a "build system" for assets.
+//! When an artist adds a new asset to the project, the asset processing system will automatically perform the specified processing steps on the asset.
```

Comment:
```suggestion
//! When an artist adds a new asset to the project, the asset processing system will automatically perform the specified[1] processing steps on the asset.
```

[1]: Specified where?

Reply:
Agreed, this could use some more detailed explanation of the actual workflow.